taskcluster/ci/maybe-nightly/kind.yml

```diff
@@ -0,0 +1,47 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+transforms:
+    - ac_taskgraph.transforms.maybe_nightly:transforms
+    - taskgraph.transforms.job:transforms
+    - taskgraph.transforms.task:transforms
+
+task-defaults:
+    run-on-tasks-for: [github-push]
+    run-on-git-branches: ['main$']
+    treeherder:
+        kind: other
+        platform: release/opt
+        tier: 3
+        symbol: maybe-nightly
```

Comment:
Nit: I'm unsure about the `maybe-nightly` name. In the current implementation, the task only runs when we're sure we want to trigger a new nightly. Thus `maybe` can be misleading. How about we call it something like `trigger-new-nightly`?

This comment applies to the treeherder symbol as well as the task kind and the transform.

Reply:
Renamed to trigger-nightly